verify.py.in

```diff
@@ -30,13 +31,31 @@ def check_vacuity(line, passed_set, failed_set):
         and m.group('result') == 'failed'):
         failed_set.add(debugInfo.strip())
 
+def generate_cex_exe(args, input_file, output_file):
+    cmd = [os.path.join(SEAHORN_ROOT, 'bin', 'sea'),
```

Comment:
I don't see how this is workable. Once the harness is produced, it has to be linked with the original input. The users (us) will want to regenerate original input by changing it and running with the same harness. 

The current solution requires re-running build and verification every time the harness is changed.

Let's come up with a better solution and then figure out how to implement it.